composer.json

```diff
@@ -19,44 +19,40 @@
     ],
     "minimum-stability": "dev",
     "require": {
-        "php": ">=5.4",
-        "yiisoft/yii2": "~2.0.13",
+        "ext-mbstring": "*",
+        "php": "^7.2 || ^8.0",
+        "yiisoft/yii2": "~2.0.16",
         "yiisoft/yii2-bootstrap": "~2.0.0",
-        "phpdocumentor/reflection": "^3.0.1",
-        "phpdocumentor/reflection-docblock": "^2.0.4",
-        "nikic/php-parser": "^1.0",
-        "cebe/js-search": "~0.9.3",
-        "cebe/markdown": "~1.0.0 | ~1.1.0 | ~1.2.0",
-        "cebe/markdown-latex": "~1.0",
-        "scrivo/highlight.php": "~9.13"
+        "phpdocumentor/reflection": "dev-access-phpparser-node",
```

Comment:
The reason I didn't open it is I doubt they will accept it or it can take a while. I guess we have to use fork for now and periodically sync it with`5.x` branch. Let's try anyway.

https://github.com/phpDocumentor/Reflection/pull/225

Reply:
`repositories` is root-only setting. It won't work if you just call `composer require yiisoft/yii2-apidoc` in your project.

Reply:
@rob006 Should I publish it to Packagist then and specify fork without "repositories"? Or maybe you want to suggest another alternative?

Reply:
I'm not really sure, but publishing fork and mentioning forked package in `replace` section (like [here](https://github.com/composer/package-versions-deprecated/blob/1927b0a81e48643ca45c4a55b34a7645b04f3acc/composer.json#L20-L22)) seems to be the only option. But still this could mess up with dependencies if someone is using reflection for other things in the project, since your changes are not compatible with original reflection package.